client.go

```diff
@@ -0,0 +1,67 @@
+package litefs
+
+import (
+	"context"
+	"net/http"
+)
+
+// SubscribeEvents subscribes to events from the default (localhost:20202)
+// LiteFS node.
+func SubscribeEvents() EventSubscription {
+	return DefaultClient.SubscribeEvents()
+}
+
+// MonitorPrimary monitors the primary status of the LiteFS cluster via the
+// default (localhost:20202) LiteFS node's event stream.
+func MonitorPrimary() PrimaryMonitor {
+	return DefaultClient.MonitorPrimary()
+}
```

Comment:
I don't think these add much value since they're one-liners. I think you can just remove them.